kornia/feature/lightglue.py

```diff
@@ -39,11 +39,13 @@
 
 
 def math_clamp(x, min_, max_):  # type: ignore
-    return max(min(x, min_), min_)
+    """Clamp a value to lie within [min, max]."""
```

Comment:
i think function like these one, shouldn't be a public one, so isn't required docstrings... we shoudl adopt over the library the use of "private" functions, instead of have to document everthing for the newones 

Most of these aux functions for models, or bigger piece of code, shouldn't be public for kornia community, if the community are using, we should make it public and then take more care over it -- since it's kinda hard re-review everthing now, i advocate for us do it in the newer code, as well, when activating this doc rules... 

but because of it being a break change idk... maybe we should only do it for the methods and classes of backbones of some model we have, and for the new codes... cc @edgarriba

Reply:
make total sense. How can we enforce in the ci to not check certain methods being `def _private` or coming from a `class _Private` ?

Reply:
The D10* rules only mention public methods/functions so any private ones shouldn't be enforced.